src/js/util.js

```diff
@@ -17,5 +17,69 @@ define(function (require, exports) {
   exports.track = function(event, data) {
     Intercom('trackEvent', event, data);
   };
+  
+  // Take an array of URLs and return a template of the form
+  // `//{s}.foo.bar/tiles/baz` and an array of substitutions for `s` that will
+  // generate the original URLs
+  exports.templatizeURLs = function templatizeURLs(urls) {
```

Comment:
Is this from an existing library or written from scratch? Can we instead use the built-in leaflet tools (eg http://leafletjs.com/reference.html#tilelayer-subdomains)? If not, we should open a ticket to write tests for this and/or write a couple quick tests to prevent regressions.